gen_cert.py

```diff
@@ -85,14 +85,20 @@ def get_cert_date(
     - If using the "ROLLING" certs feature, use today's date
     - If all else fails use 'configured_date_parameter' for date
     """
+    utc_zone = get_timezone('UTC')
```

Comment:
'cause this is just the initial timezone so all dates start on equal footing.
the real timezone gets applied later programmatically.